enterprise/backend/test/metabase_enterprise/serialization/api_test.clj

```diff
@@ -291,8 +291,19 @@
                              "field_values"    false
                              "secrets"         false
                              "success"         false
-                             "error_message"   #"clojure.lang.ExceptionInfo: Exception extracting Card.*"}
-                            (-> (snowplow-test/pop-event-data-and-user-id!) first :data)))))
+                             "error_message"   #"(?s)Exception extracting Card \d+ .*"}
+                            (-> (snowplow-test/pop-event-data-and-user-id!) first :data))))
+
+                  (testing "Full stacktrace"
+                    (binding [api.serialization/*additive-logging* false]
+                      (let [res (mt/user-http-request :crowberto :post 500 "ee/serialization/export"
+                                                      :collection (:id coll) :data_model false :settings false
+                                                      :full_stacktrace true)
+                            log (slurp (io/input-stream res))]
+                        (is (< 200
```

Comment:
this assertion is a bit generic :D, can we be more specific and find stacktrace from the log?

Reply:
😁 log becomes 536 lines instead of 4 lines, so it's not _that_ generic. 

Any ideas how to catch "long stacktrace" better? I mean it superficially looks like the same as short error message, but is just filled with lots of lines inside.

Reply:
I think this is fine. If there are tons of non stacktrace lines, e.g. a monster message or data map that's still a big problem.

Reply:
Nah, it's quite controlled: there is a line for every entity being exported (3 in total), and then the gigantic stacktrace. I can add a test that previous log is less than 10 lines if y'all so concerned. 😁

Reply:
Nah I was just advocating why I like this dumb assertion.